Automattic/wp-calypso
# Open links for generated response in new tab

Links clicked in the chat GPT-generated content can break the user experience and navigation.
This PR will ensure that the links clicked are opened in new tab and won't be that disruptive for the customer.

Related to #
8119-gh-Automattic/dotcom-forge

## Proposed Changes

* Listen to content link click, and open the link in new tab

## Why are these changes being made?


* The customer can potentially end up on a page where the help center is not available
* It can interfere with the navigation

## Testing Instructions
- Login as a free user
- Open the help center and click "Still need help"
- For subject enter "Register domain" and for content enter "how do I register domain"
- Check if links clicked in the generated content are opened in a new tab



*

## Pre-merge Checklist
